examples/Demo/FluentUI.Demo.Client/Documentation/Components/Splitter/MultiSplitter.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Splitter
+route: /Splitter
+---
+
+# Splitter
+
+The **MultiSplitter** splits the page into multiple panels and allows the user to control the page layout.
+
+You can include as many **Panes** as you like in a container.
+By default, the panes are resizable, but the `Resizable` parameter can be used to block this functionality.
+The initial `Size` is in percent or pixels and has a `Min` and `Max` value.  
+A Panel can be fully collapsed by setting the `Collapsible` parameter to `true`.
+
+The `OnResize` event returns the new size as a percentage.
+
+## Example
+
+In this example, each time a pane is resized, a trace is written to the console.
+
+{{ MultiSplitterDefault }}
+
+## Restrictions
+
+This component must be able to communicate between the DOM (browser) and the .NET runtime.
+Consequently, this component **requires an Interactive mode** (interactive SSR, interactive WASM).
```

Comment:
Please rephrase and use the same terms Blazor uses: Interactive Server, Interactive WebAssembly or Auto. Calling it SSR might cause confusion.